tests/app/celery/test_scheduled_tasks.py

```diff
@@ -882,17 +874,57 @@ def test_zendesk_new_email_branding_report_calculates_last_weekday_correctly(
     org_1.email_branding_pool = [email_brand_1]
     notify_db_session.commit()
 
+    mocker.patch("app.celery.scheduled_tasks.NotifySupportTicket", wraps=NotifySupportTicket)
     mock_send_ticket = mocker.patch("app.celery.scheduled_tasks.zendesk_client.send_ticket_to_zendesk")
 
     with freeze_time(freeze_datetime):
         zendesk_new_email_branding_report()
 
-    assert mock_send_ticket.call_count == 1
-    ticket = mock_send_ticket.call_args_list[0][0][0]
-    assert expected_last_day_string in ticket.request_data["ticket"]["comment"]["html_body"]
+    # Make sure we've built a NotifySupportTicket with the expected params, and passed that ticket to the zendesk client
+    assert expected_last_day_string in mock_send_ticket.call_args_list[0][0][0].message
 
 
 def test_zendesk_new_email_branding_report_does_not_create_ticket_if_no_new_brands(notify_db_session, mocker):
     mock_send_ticket = mocker.patch("app.celery.scheduled_tasks.zendesk_client.send_ticket_to_zendesk")
     zendesk_new_email_branding_report()
     assert mock_send_ticket.call_args_list == []
+
+
+def test_check_for_low_available_inbound_sms_numbers_logs_zendesk_ticket_if_too_few_numbers(notify_db_session, mocker):
+    mocker.patch.dict("app.celery.scheduled_tasks.current_app.config", {"LOW_INBOUND_SMS_NUMBER_THRESHOLD": 10})
```

Comment:
We have context managers to set a either a single config value or multiple in tests, for example
https://github.com/alphagov/notifications-api/blob/ac7791692ad488e99a92ecea12144c574f0598bb/tests/app/broadcast_message/test_utils.py#L338-L339

I'm trying to remember if there was a particular reason why we always tend to use that way of doing things instead of mocking the config

Reply:
Cool, will switch 👍  They both do the same thing, looks like we've just made our own mocker.patch.ditch 😅

Reply:
Patched in [28c348f](https://github.com/alphagov/notifications-api/pull/3656/commits/28c348ff4c00ecff450531c6076cd0e045142b69)